.travis_scripts/build_all

```diff
@@ -29,7 +30,7 @@ echo ""
 echo "Configuring conda."
 conda config --add channels conda-forge
 conda config --set show_channel_urls true
-conda install --yes --quiet conda-forge-ci-setup=1.* conda-forge-pinning networkx conda-build=3.16
+conda install --yes --quiet conda-forge-ci-setup=2.* conda-forge-pinning networkx conda-build
```

Comment:
Bumping the `conda-forge-ci-setup` version constraint seems reasonable. Would submit a PR and request core's feedback to see if there is something else we are missing or not.

Reply:
Thanks: https://github.com/conda-forge/staged-recipes/pull/7429